frontend/app/src/components/history/events/HistoryEventsTable.vue

```diff
@@ -233,6 +234,10 @@ function forceRedecode(): void {
   set(deleteCustom, false);
   set(redecodePayload, undefined);
 }
+
+watch(() => props.groups.data, (newRows) => {
+  emit('current-page-events', newRows);
+}, { immediate: true });
```

Comment:
Why do you need to emit it from here, while the data is passed from parent, meaning you can get it directly in parent